actions/views/root.js

```diff
@@ -48,20 +48,19 @@ export function unregisterPluginTranslationsSource(pluginId) {
 }
 
 export function loadTranslations(locale, url) {
-    return (dispatch) => {
+    return async (dispatch) => {
         const translations = {};
         Object.values(pluginTranslationSources).forEach((pluginFunc) => {
             Object.assign(translations, pluginFunc(locale));
         });
+        let localeTranslations = en;
+
+        // Need to go to the server for languages other than English
 
-        // No need to go to the server for EN
-        if (locale === 'en') {
-            copyAndDispatchTranslations(dispatch, translations, en, locale);
-            return;
+        if (locale !== 'en') {
+            localeTranslations = await Client4.getTranslations(url);
```

Comment:
Since this can throw an error, we should wrap it in a try catch.
```
if (locale !== 'en') {
	try {
	  	const serverTranslations = await Client4.getTranslations(url);
		Object.assign(translations, serverTranslations);
	} catch (error) {
	  	console.error(error); //eslint-disable-line no-console
	}
}
```